lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -406,6 +417,13 @@ dummy 1.2.3
             packageName: 'casey/just',
             depName: 'just',
           },
+          {
+            currentValue: '1.31.2+k3s1',
+            datasource: 'github-releases',
+            packageName: 'k3s-io/k3s',
+            depName: 'k3s',
+            extractVersion: '^v(?<version>\\S+)',
```

Comment:
drop, not necessary. please also check others. this is mostly not required

Reply:
so just the `currentValue` is sufficient? sorry, just trying to mimic what's already done. I'm happy to include the update :)

Reply:
yes, we need to check and remove from existing too in a separate PR.

Reply:
ok, removed